### DISCUSSION TITLE
Agent Transfer Option

### DISCUSSION THREAD

**Discussion by mahfuzplabon** [chatwoot/chatwoot#2529] [Q&A]
Hello,

Is there any option to transfer chat from One Agent to Another by the agent himself ??

I mean like if Agent 1 is unable to do the thing, he can transfer the chat to Agent 2, is it already there in Chatwoot ??

Regards.

**Comment by sojan-official** [comment]
Yes , you can find the option to transfer to another agent in the conversation sidebar

<img width="1818" alt="Screenshot 2021-06-28 at 6 52 11 PM" src="https://user-images.githubusercontent.com/73185/123643620-31cea700-d842-11eb-835a-c5e04fd60549.png">
